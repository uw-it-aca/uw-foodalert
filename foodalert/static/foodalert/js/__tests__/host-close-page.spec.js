"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import ClosePage from '../pages/host/close-page.vue';

describe('Host Close Page', function () {
        test('renders correctly', function () {
            var wrapper = mount(ClosePage, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})