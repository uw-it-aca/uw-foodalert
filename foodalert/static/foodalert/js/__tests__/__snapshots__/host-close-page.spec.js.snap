// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Host Close Page renders correctly 1`] = `
<div
  class="page"
>
  <div
    id="standard-notification"
  >
    <div
      class="collapse"
      id="notif-container"
      style="display: none;"
    >
      <div
        class="text-center py-3 container-fluid"
        style="background-color: rgb(0, 112, 201);"
      >
        <div
          class="container"
        >
          <div
            class="row justify-content-center"
          >
            <div
              class="col-md-8"
            >
              <div
                class="h5 text-white pt-2"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <main
    class="page-content pb-2"
  >
     
    <div
      id="standard-container"
    >
      <h1
        class="pt-md-5 pt-3 pb-2"
        id="standard-heading"
      >
        
        Your food cannot be shared with Hungry Husky
    
      </h1>
       
      <div
        id="standard-body"
      >
        <p>
          You indicated that your food is homemade, which makes it
        ineligible to be shared with the public via Hungry Husky.
        </p>
         
        <p>
          You are still able to share it with your colleagues in your
        office. If you have questions about how to share food with Hungry
        Husky, please email
        
          <a
            href="mailto:hungryhusky@uw.edu"
          >
            hungryhusky@uw.edu
          </a>
          .
        And thanks for thinking of Hungry Husky!
        </p>
      </div>
       
    </div>
  </main>
   
  <footer
    class="text-center"
    id="relative-footer"
  >
    <a
      href="mailto:help@uw.edu?subject=Hungry Husky support"
    >
      
        Contact support
      
    </a>
     
    <p>
      Â© 2019 University of Washington
    </p>
  </footer>
</div>
`;
